dataSource1/src/main/java/QuizGateway.kt

```diff
@@ -0,0 +1,4 @@
+interface QuizGateway {
```

Comment:
Create separate gateway for token as questions and  token are logically different things and cannot be provided by same datasource